docs/developers/deploy-on-polygon.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Deploying the LSP on Polygon
-sidebar_label: Deploying the LSP on Polygon
+title: Deploying an LSP Contract
```

Comment:
Might be good to change the filename as well, since this controls the URL as well.